src/service.js

```diff
@@ -311,7 +310,7 @@ const convertSpaceToFile = async (id, query, headers) => {
     // just include the token string and not the bearer prefix
     token = token.substring(7);
   }
-  const params = query;
+  const params = body;
```

Comment:
Do we need to reassign this, or can we simply go with `body` throughout the method?

Reply:
It is because of the assignment to the property of the function parameter